script.js

```diff
@@ -1,406 +1,399 @@
-      '2 medium cloves garlic, minced (about 2 teaspoons)',
-      '2 tablespoons vegetable or canola oil',
-      '1 whole flap meat steak, 2 to 2 1/2 pounds',
-      'Kosher salt'
+      "1 tablespoon whole black peppercorns, toasted",
+      "1 teaspoon coriander seed, toasted",
+      "1 teaspoon fennel seed, toasted",
+      "1 teaspoon cumin pods, toasted",
+      "1 teaspoon red pepper flakes",
+      "1/2 teaspoon dried oregano",
+      "2 medium cloves garlic, minced (about 2 teaspoons)",
+      "2 tablespoons vegetable or canola oil",
+      "1 whole flap meat steak, 2 to 2 1/2 pounds",
+      "Kosher salt",
     ],
-    source: 'Serious Eats',
+    source: "Serious Eats",
     totalTime: 240,
-    url: 'http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html',
-    image: './recipe-images/grilled.jpg'
-  }
+    url: "http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html",
+    image: "./recipe-images/grilled.jpg",
+  },
+  {
+    name: "Gnocchi Alla Sorrentina",
+    cuisineType: ["italian"],
+    ingredients: [
+      "2 tablespoons extra-virgin olive oil",
+      "1 clove garlic, lightly crushed and peeled",
+      "Generous pinch dried red pepper flakes",
+      "4 cups passata or tomato purée",
+      "Fine salt",
+      "Two sprigs basil",
+      "1 ball mozzarella, diced",
+      "16 ounces potato gnocchi, homemade or store-bought",
+      "Freshly grated Parmigiano-Reggiano",
+    ],
+    source: "Domenica Marchetti",
+    totalTime: 60,
+    url: "https://www.themediterraneandish.com/gnocchi-alla-sorrentina/",
+    image: "./recipe-images/gnochi.jpeg",
+    favorite: true,
+  },
 ]
+
+//Function to show the recipes
+const showRecipes = (recipesToShow) => {
+  container.innerHTML = ""
+
+  recipesToShow.forEach((recipe) => {
+    // Capitalize the first letter of cuisineType
+    const capitalizedCuisineType =
+      recipe.cuisineType[0].charAt(0).toUpperCase() +
+      recipe.cuisineType[0].slice(1)
+
+    container.innerHTML += `
+    <div class="recipe-card">
+      <img src="${recipe.image}"/>
+      <div class="recipe-text">
+        <h3>${recipe.name}</h3>
+        <p><b>Cuisine:</b> ${capitalizedCuisineType}</p>
+        <p><b>Cooking time:</b> ${recipe.totalTime} minutes</p>
+        <p><b>Ingredients:</b></p>
+        <ul>
+          ${recipe.ingredients
+            .map((ingredient) => `<li>${ingredient}</li>`)
+            .join("")}
+        </ul>
+        <p class="recipe-link"><a href="${
+          recipe.url
+        }" target="_blank">Link to cooking method</a>
+        </p>
+      </div>
+    </div>
+    `
+  })
+}
+
+//Initial rendering of all recipes
+showRecipes(recipes)
+
+//Function random
+const getRandomValue = () => {
+  container.innerHTML = ""
+
+  recipes[Math.floor(Math.random() * recipes.length)]
+  const recipe = recipes[Math.floor(Math.random() * recipes.length)]
+  container.innerHTML += `
```

Comment:
Here you could have called the showRecipes function and connected it to the result of your getRandomValue function, no need to rewrite the same innerHTML again :)